src/stdatamodels/jwst/datamodels/tests/test_integration.py

```diff
@@ -101,16 +97,16 @@ def test_schema_refs_base(datamodel_schema_file):
     def cb(subschema, path, combiner, ctx, recurse):
         if not isinstance(subschema, dict):
             return
-        if 'id' not in subschema:
+        if "id" not in subschema:
             return
-        ctx['ids'].add(subschema['id'])
+        ctx["ids"].add(subschema["id"])
 
     seen_ids = set()
-    stdatamodels.schema.walk_schema(schema, cb, ctx={'ids': seen_ids})
+    stdatamodels.schema.walk_schema(schema, cb, ctx={"ids": seen_ids})
 
-    if 'http://stsci.edu/schemas/jwst_datamodel/core.schema' in seen_ids:
-        assert 'http://stsci.edu/schemas/jwst_datamodel/referencefile.schema' not in seen_ids
-    elif 'http://stsci.edu/schemas/jwst_datamodel/referencefile.schema' in seen_ids:
-        assert 'http://stsci.edu/schemas/jwst_datamodel/core.schema' not in seen_ids
+    if "http://stsci.edu/schemas/jwst_datamodel/core.schema" in seen_ids:
+        assert "http://stsci.edu/schemas/jwst_datamodel/referencefile.schema" not in seen_ids
+    elif "http://stsci.edu/schemas/jwst_datamodel/referencefile.schema" in seen_ids:
+        assert "http://stsci.edu/schemas/jwst_datamodel/core.schema" not in seen_ids
     else:
-        assert False
+        assert False  # noqa: B011
```

Comment:
Could this be replaced with a `raise AssertionError` as the rule suggests?

Or I guess it's fine to wait until the "fix later" assert removes happen, so nevermind

Reply:
I think `assert False` is more readable. Perhaps `B011` should be disabled for tests since the description of [why it's bad](https://docs.astral.sh/ruff/rules/assert-false/) doesn't apply to tests:
> Python removes assert statements when running in optimized mode (python -O), making assert False an unreliable means of raising an AssertionError.

Reply:
I'm fine with disabling it for tests.  TBH I think I didn't realize I was in a test when I made the comment

Reply:
disabled in 4090a783ae3eea9c6ec859b30f74ca0b8ad62a00